app/assets/javascripts/components/students_card.es6.jsx

```diff
@@ -0,0 +1,73 @@
+class StudentsCard extends React.Component {
+
+  static get propTypes() {
+    return {
+      firstName: React.PropTypes.string.isRequired,
+      lastName: React.PropTypes.string.isRequired,
+      birthday: React.PropTypes.string.isRequired,
+      age: React.PropTypes.string.isRequired,
+      phoneNumber: React.PropTypes.string.isRequired,
+      email: React.PropTypes.string.isRequired,
+      school: React.PropTypes.string.isRequired,
+      status: React.PropTypes.string.isRequired,
+    };
+  }
+
+  static get defaultProps() {
+    return {
+      firstName: '',
+      lastName: '',
+      birthday: '',
+      age: '',
+      phoneNumber: '',
+      email: '',
+      school: '',
+      status: '',
+    };
+  }
+
+  get styles() {
+    return {
+      container: {
+        display: 'flex',
+        flexDirection: 'row',
+        position: 'relative',
+        height: '312px',
+        backgroundColor: 'white',
+        border: '1px solid',
+        borderColor: '#e5e6e9 #dfe0e4 #d0d1d5',
```

Comment:
I'm creepin but these colors in a constants file should be put in a constants file. This is replicated [5 times elsewhere](https://github.com/calblueprint/edge-yl/search?utf8=%E2%9C%93&q=%23e5e6e9).